src/devices/MemoryLCD/LSxxxB7DHxx.cs

```diff
@@ -0,0 +1,300 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Gpio;
+using System.Device.Spi;
+using System.Drawing;
+using System.Linq;
+using System.Threading;
+
+namespace Iot.Device.MemoryLcd
+{
+    /// <summary>
+    /// Memory LCD
+    /// </summary>
+    public abstract class LSxxxB7DHxx : IDisposable
+    {
+        #region Screen specification
+
+        /// <summary>
+        /// Memory LCD width pixels
+        /// </summary>
+        public abstract int PixelWidth { get; }
+
+        /// <summary>
+        /// Memory LCD height pixels
+        /// </summary>
+        public abstract int PixelHeight { get; }
+
+        /// <summary>
+        /// Memory LCD bytes per line
+        /// </summary>
+        public int BytesPerLine { get; }
+        #endregion
+
+        #region GPIO config
+
+        /// <summary>
+        /// Chip select signal
+        /// </summary>
+        private readonly int _scs;
+
+        /// <summary>
+        /// Display ON/OFF signal<br/>The display ON/OFF signal is only for display. Data in the memory will be saved at the time of ON/OFF.<br/>When it’s "H", data in the memory will display, when it’s "L", white color will diaplay and data in the memory will be saved.
+        /// </summary>
+        private readonly int _disp;
+
+        /// <summary>
+        /// External COM inversion signal input (H: enable)<br/>When EXTMODE is "Lo", connect the EXTCOMIN to VSS.
+        /// </summary>
+        private readonly int _extcomin;
+        #endregion
+
+        #region  Delay constants for LCD timing
+
```

Comment:
Please add a link on the documentation where you picked the timing.
Also, if you need µ seconds, you can if you're using the Delay Helper. Would be much better than the traditional Thread.Sleep